srcpkgs/OpenRCT2/template

```diff
@@ -2,33 +2,42 @@
 # based on https://raw.githubusercontent.com/AluisioASG/void-packages/openrct2/srcpkgs/OpenRCT2/template
 # and https://github.com/void-linux/void-packages/issues/1014#issuecomment-417372421
 pkgname=OpenRCT2
-_objects_version=1.0.15
-_titles_version=0.1.2
-version=0.2.6
-revision=3
+_objects_version=1.0.18
+_titles_version=0.1.2c
+version=0.3.2
+revision=1
 build_style=cmake
-configure_args="$(vopt_if multiplayer "" "-DDISABLE_HTTP_TWITCH=1 -DDISABLE_NETWORK=1")"
+configure_args="
+ -DOPENRCT2_VERSION_TAG=${version}
+ -DDOWNLOAD_TITLE_SEQUENCES=0
+ -DDOWNLOAD_OBJECTS=0
+ $(vopt_if multiplayer '' '-DDISABLE_NETWORK=1')
+ $(vopt_if scripting '-DENABLE_SCRIPTING=1' '')"
 hostmakedepends="pkg-config unzip"
 makedepends="SDL2-devel fontconfig-devel freetype-devel libzip-devel
  libpng-devel speexdsp-devel jansson-devel icu-devel zlib-devel
- $(vopt_if multiplayer libcurl-devel) $(vopt_if multiplayer libressl-devel)"
+ json-c++>=3.9.1_2
```

Comment:
Packages are always built with the versions of dependencies specified in the templates, so you don't need to specify a version here. This is probably what broke the build.